packages/shared/src/components/sidebar/MyFeedButton.tsx

```diff
@@ -66,7 +66,7 @@ const getAnalyticsEvent = (
 ): Partial<AnalyticsEvent> => ({
   event_name: eventName,
   target_type: 'my feed button',
-  target_id: 'sidebar',
+  target_id: 'my-feed',
```

Comment:
Actually sidebar was better, I think.
This is why I resolved my comment, didn't want you to change after our discussion

Reply:
Oh sorry I already changed it yesterday, will revert this one :)